java-manta-client/pom.xml

```diff
@@ -170,6 +170,20 @@
             <version>${dependency.urlbuilder.version}</version>
         </dependency>
 
+        <!-- object cloning library for memoization of encryption state -->
```

Comment:
I don't think it's within the scope of this change to change the behavior of serialization, in fact I've opted to shade these dependencies specifically to avoid conflict with kryo. That said it makes sense to include a file with serialized objects in it so I'll create an issue to figure out what the content would be and how the tests would be designed.